simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/Station.java

```diff
@@ -48,28 +44,41 @@
     private final StationMessageInbox stationMessageInbox = new StationMessageInbox();
 
     /**
-     * Create a station using {@link SimulatorConfiguration.StationConfiguration} and defaultHeartBeatIntervalSec.
+     * Create a station using {@link SimulatorConfiguration.StationConfiguration}, {@link SimulatorConfiguration.WebSocketConfiguration} and defaultHeartBeatIntervalSec.
      *
-     * @param configuration station configuration
+     * @param stationConfiguration station configuration
+     * @param socketConfiguration socket configuration
      */
-    public Station(SimulatorConfiguration.StationConfiguration configuration) {
-        this(configuration, DEFAULT_HTTP_CLIENT);
-    }
+    public Station(SimulatorConfiguration.StationConfiguration stationConfiguration, SimulatorConfiguration.WebSocketConfiguration socketConfiguration) {
+        OkHttpClient.Builder httpClientBuilder = new OkHttpClient.Builder()
+                .addInterceptor(new LoggingInterceptor())
+                .addNetworkInterceptor(new LoggingInterceptor())
+                .pingInterval(10, TimeUnit.SECONDS);
+
+        if (socketConfiguration != null) {
```

Comment:
I'd add another constructor with only StationConfiguration param, which would be calling this one and passing new WebSocketConfiguration()